lib/mix/lib/mix/cli.ex

```diff
@@ -9,6 +9,7 @@ defmodule Mix.CLI do
     Mix.Local.append_paths
 
     if System.get_env("MIX_QUIET"), do: Mix.shell(Mix.Shell.Quiet)
+    if System.get_env("MIX_DEBUG"), do: Mix.debug(true)
```

Comment:
Where should I document this? I found that `MIX_QUIET` is described in the man page, 
so it would be the right place to add info about this right?